docs/knative_e2e_tests.md

```diff
@@ -11,10 +11,6 @@ steps detailed here.
 - (OPTIONAL) Deploy a local Kubernetes test environment with Knative Serving and
   Kourier if needed:
 
-```bash
-make local-setup
-```
-
```

Comment:
Is the rest of the doc useful and still functional? I see some outdated tag refs. 
If not either we update it or remove it completely for now.

Reply:
You're right. I'll review it, so please give me some time. 
If possible, I'd like to keep the document in a functional state.

Reply:
@skonto 

It didn’t work on Knative `v0.9.0`, but the tests passed on `knative-v1.17.0`.  
However, since the ingress tests were removed, I also removed the corresponding wording.

- FYI: https://github.com/knative/serving/pull/8968